src/Form/FastlySettingsForm.php

```diff
@@ -14,219 +14,220 @@
-      // Hide while no API key is set.
-      '#states' => [
-        'invisible' => [
-          'input[name="api_key"]' => ['empty' => TRUE],
-        ],
-      ],
-      '#prefix' => '<div id="edit-service-wrapper">',
-      '#suffix' => '</div>',
-    ];
-
-    $form['purge_method'] = [
-      '#type' => 'radios',
-      '#title' => $this->t('Purge method'),
-      '#description' => $this->t("Switch between Fastly's Instant-Purge and Soft-Purge methods."),
-      '#default_value' => $config->get('purge_method') ?: self::FASTLY_INSTANT_PURGE,
-      '#options' => [
-        self::FASTLY_INSTANT_PURGE => $this->t('Use instant purge'),
-        self::FASTLY_SOFT_PURGE => $this->t('Use soft purge'),
-      ],
-    ];
-
-    $form['purge'] = [
-      '#type' => 'details',
-      '#title' => $this->t('Soft purge options'),
-      '#open' => TRUE,
-      '#states' => [
-        'invisible' => [
-          ':input[name="purge_method"]' => ['value' => self::FASTLY_INSTANT_PURGE],
-        ],
-        'required' => [
-          ':input[name="purge_method"]' => ['value' => self::FASTLY_SOFT_PURGE],
-        ],
-      ],
-    ];
-
-    $form['purge']['stale_while_revalidate_value'] = [
-      '#type' => 'number',
-      '#title' => $this->t('Stale while revalidate'),
-      '#description' => $this->t('The number in seconds to show stale content while cache revalidation.'),
-      '#default_value' => $config->get('stale_while_revalidate_value') ?: 604800,
-    ];
-
-    $form['purge']['stale_if_error'] = [
-      '#type' => 'checkbox',
-      '#title' => $this->t('Stale if error'),
-      '#description' => $this->t("Activate the stale-if-error tag for serving stale content if the origin server becomes unavailable."),
-      '#default_value' => $config->get('stale_if_error'),
-    ];
-
-    $form['purge']['stale_if_error_value'] = [
-      '#type' => 'number',
-      '#description' => $this->t('The number in seconds to show stale content if the origin server becomes unavailable.'),
-      '#default_value' => $config->get('stale_if_error_value') ?: 604800,
-      '#states' => [
-        'invisible' => [
-          ':input[name="stale_if_error"]' => ['checked' => FALSE],
-        ],
-        'required' => [
-          ':input[name="stale_if_error"]' => ['checked' => TRUE],
-        ],
-      ],
-    ];
-
-    return parent::buildForm($form, $form_state);
-  }
-
-  /**
-   * Handles changing the API key.
-   */
-  public function updateServices($form, FormStateInterface $form_state) {
-    return $form['service_id'];
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public function validateForm(array &$form, FormStateInterface $form_state) {
-    if (!$this->api->validatePurgeCredentials($form_state->getValue('api_key'))) {
-      $form_state->setErrorByName('api_key', $this->t('Invalid API token. Make sure the token you are trying has at least <em>global:read</em>, <em>purge_all</em>, and <em>purge_all</em> scopes.'));
+    /**
```

Comment:
Needs adjustment for coding standards -- this will help keep the diff sane and make review easier.